### DISCUSSION TITLE
Kernel crash while executing .position for scene objects

### DISCUSSION THREAD

**Discussion by Priyanshjain24** [NVlabs/sionna#450] [Q&A] [answered]
I am trying to get the position of scene object from an example ray tracing scenario. Each time I execute `obj.position` the kernel crashes with the error - `14:45:17.455 [error] Disposing session as kernel process died ExitCode: undefined, Reason: `. I ran the same code in a python file, but I got an error `F tensorflow/core/framework/tensor.cc:847] Check failed: IsAligned() ptr = 0x56dfdee42da0` `Aborted (core dumped)`. I installed sionna by `pip install sionna` in virtual environment. I am getting the same issues both on my local system as well as Google Colab. Any help would be appreciated. 

Code that I used:
```
import os
gpu_num = "" # Use 0,1,2 to use different GPU
os.environ["CUDA_VISIBLE_DEVICES"] = f"{gpu_num}"
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'

import sionna
import numpy as np
import tensorflow as tf
tf.random.set_seed(1)
from sionna.rt import load_scene, PlanarArray, Transmitter, Receiver, Camera

scene = load_scene(sionna.rt.scene.simple_reflector)
obj = scene.get('reflector')
print("Object: ", obj)
print("Velocity: ", obj.velocity)
print("Position: ", obj.position)
```

**Comment by SebastianCa** [comment]
Hi @Priyanshjain24,

this seems to be related to #449. We'll look into the issue and keep you updated.

**Reply by merlinND** [reply, marked as answer]
Closing as duplicate of #449, @Priyanshjain24 please subscribe to that issue for updates.
